src/helpers/cli.js

```diff
@@ -1,9 +1,28 @@
 import yargs from "yargs";
+import { init as generate } from "./../generate";
+import { init as draft } from "./../draft";
+import { init as fetch } from "./../fetch";
```

Comment:
Can this be renamed so it's not confused with the native `fetch` call that's available in browsers?